chopper/lib/src/request.dart

```diff
@@ -1,14 +1,14 @@
 import 'dart:async';
 
-import 'package:chopper/src/extensions.dart';
+import 'package:chopper/chopper.dart';
```

Comment:
Why? Only the extension is needed. I don't see a point in importing the whole package for it.

Reply:
Probably auto import from IDE😓